### DISCUSSION TITLE
Move just one window of an app to another screen

### DISCUSSION THREAD

**Discussion by chodorowicz** [rxhanson/RectanglePro-Community#123] [Q&A] [answered]
Hey! 🙂
We have an action to move apps between displays, but they move all windows of an app. Could we add an action that moves just the active window to another display and not all apps?
Thanks!

**Comment by rxhanson** [comment, marked as answer]
I think you might have skipped over the single window options, straight to the multi-window ones. Are these what you are looking for?

![image](https://user-images.githubusercontent.com/13651296/192162537-ac3f7298-4cbc-4fa3-8886-19fb81f23c6d.png)

**Reply by chodorowicz** [reply]
Yes, of course, 🤦  Thanks so much for pointing that out. Cheers 🙇‍♂️
